packages/lib/import-enex-md-gen.ts

```diff
@@ -741,7 +755,19 @@ function enexXmlToMdArray(stream: any, resources: ResourceEntity[]): Promise<Ene
 				section.lines.push(newSection);
 				section = newSection;
 			} else if (isBlockTag(n)) {
-				section.lines.push(BLOCK_OPEN);
+				const isTodosList = cssValue(this, nodeAttributes.style, '--en-task-group') === 'true';
+				if (isTodosList) {
+					const todoGroup = cssValue(this, nodeAttributes.style, '--en-id');
+					section.lines.push(BLOCK_OPEN);
+					tasks.filter(t => t.groupId === todoGroup)
+						.forEach(t => {
```

Comment:
We have a rule to disallow forEach (more info [here](https://github.com/github/eslint-plugin-github/blob/main/docs/rules/array-foreach.md)) so I think CI will fail here. Please just replace it with `for (... of ...)`

Reply:
fixed in commit.